[Zaraz] New attributes input on the dash (#5717)
cloudflare/cloudflare-docs: content/zaraz/reference/properties-reference.md

_URL host_ | String | Returns the domain (that is the hostname) followed by a `:` and the port of the URL (if a port was specified).
_URL hostname_ | String | Returns the domain of the URL.
_URL origin_ | String | Returns the origin of the URL, that is its scheme, its domain and its port.
_URL password_ | String | Returns the password specified before the domain name.
_ULR pathname_ | String | Returns the path of the URL, including the initial `/`. Does not include the query string or fragment.
_URL port_ | String | Returns the port number of the URL.
_URL protocol scheme_ | String | Returns the protocol scheme of the URL, including the final `:`.
_URL query parameters_ | String | Returns search parameters provided, beginning with the leading `?` character.
_URL username_ | String | Returns the username specified before the domain name.

{{</table-wrap>}}

## Cookies

{{<table-wrap>}}

Property | Type | Description
--- | --- | ---
_Cookie: name:_ | Object | Returns cookies obtained from the browser `document`.

{{</table-wrap>}}

## Device properties

{{<table-wrap>}}

Property | Type | Description
--- | --- | ---

> _Browser engine_ | String | Returns the type of browser engine (for example, WebKit).

Reviewer: ```suggestion
_Browser engine_ | String | Returns the type of browser engine (for example, `WebKit`).
```

_Browser engine version_ | String | Returns the version of the browser’s engine.
_Browser name_ | String | Returns the browser’s name.
_Browser version_ | String | Returns the browser’s version.
_Device CPU_ | String | Returns the device’s CPU.
_Device IP address_ | String | Returns the incoming IP address.
_Device language_ | String | Returns the language used.

> _Device screen resolution_ | String | Returns screen resolution for device.

Reviewer: ```suggestion
_Device screen resolution_ | String | Returns the screen resolution of the device.
```

> _Device type_ | String | Returns the type of device used (for example, iPhone).

Reviewer: ```suggestion
_Device type_ | String | Returns the type of device used (for example, `iPhone`).
```

_Device viewport_ | String | Returns the visible web page area in user’s device.
_Operating system name_ | String | Returns the operating system.
_Operating system version_ | String | Returns the version of the operating system.
_User-agent string_	| String | Returns the browser’s user agent.

{{</table-wrap>}}

## Miscellaneous

{{<table-wrap>}}

Property | Type | Description
--- | --- | ---
_Random number_ | Number | Returns a random number unique to each request.

> _Timestamp (milliseconds)_ | Number | Returns Unix time in milliseconds.

Reviewer: ```suggestion
_Timestamp (milliseconds)_ | Number | Returns the Unix time in milliseconds.
```

> _Timestamp (seconds)_ | Number | Returns Unix time in seconds.

Reviewer: ```suggestion
_Timestamp (seconds)_ | Number | Returns the Unix time in seconds.
```
